### DISCUSSION TITLE
Docker Swarm + Traefik setup

### DISCUSSION THREAD

**Discussion by prologic** [sablierapp/sablier#451] [Q&A]
I see a [few](https://github.com/orgs/sablierapp/discussions?discussions_q=is%3Aopen+traefik+swarm) references to using sablier in a Docker Swarm environment with the Traefik reverse proxy. Would anyone already gone through all the effort successfully mind writing up a short guide on their setup? 🙏

**Comment by acouvreur** [comment]
Maybe this end to end test with docker swarm and traefik will help you:

https://github.com/sablierapp/sablier/blob/b5c68c6f1ba4c4a3906cdce95ee6d6fe24d62cef/plugins/traefik/e2e/docker_swarm/docker-stack.yml

Just change the sablier version to the latest ones :)
